app/src/androidTest/java/com/bottlerocketstudios/brarchitecture/PullRequestScreenshotTest.kt

```diff
@@ -0,0 +1,80 @@
+package com.bottlerocketstudios.brarchitecture
+
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onRoot
+import com.bottlerocketstudios.compose.pullrequest.PullRequestItemState
+import com.bottlerocketstudios.compose.pullrequest.PullRequestScreen
+import com.bottlerocketstudios.compose.pullrequest.PullRequestScreenState
+import com.bottlerocketstudios.compose.util.asMutableState
+import com.karumi.shot.ScreenshotTest
+import org.junit.Rule
+import org.junit.Test
+
+class PullRequestScreenshotTest : ScreenshotTest {
+
+    @get:Rule
+    val composeRule = createComposeRule()
+
+    @Test
+    fun pullRequestWithListItemsScreenshot() {
+        renderComponent(testList)
+        compareScreenshot(composeRule.onRoot())
+    }
+
+    @Test
+    fun pullRequestNoResultItemsScreenshot() {
+        renderComponent(emptyList)
+        compareScreenshot(composeRule.onRoot())
+    }
+
+    fun renderComponent(state: PullRequestScreenState? = null) {
```

Comment:
@Br-DiorD We should change the type of the state argument to be non nullable as seen below:

```kotlin
fun renderComponent(state: PullRequestScreenState) {
```